client/views/admin/elections/electionsAdminResults.coffee

```
@@ -83,6 +83,25 @@ Template.electionsAdminResults.helpers
       return information
   getWinner: () ->
     questionId = @_id
+    election = Election.getActive()
+    question = getQuestion(questionId)
```

This should throw an error as `questionId` is not defined right? You need to pass it in as a function argument like for example votes is in the corresponding html file in the same folder `<tr><td>Votes</td><td>{{votes _id ../_id ../../votes }}</td></tr>` and of course declare that argument in this function as well